### PR TITLE
docs(style-guide): setup, basic style guide

### PR DESCRIPTION
**Pull request by thuanpham2311**
#32

### CODE REVIEW

**Review by MiloNguyen95 — APPROVED** [review]

File: docs/style-guide.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+## Colors
+
+**css variable**
+
+```css
+:root {
+  --view-reply-text-color: rgba(61, 166, 255, 1);
+  --video-title-text-color: rgba(255, 255, 255, 1);
+  --video-bar-time-played-bg-color: rgba(255, 179, 71, 1);
+  --video-bar-bg: rgba(255, 255, 255, 0.15);
```
Comment (thuanpham2311) [inline docs/style-guide.md:23]: typo: bg -> bg-color

File: docs/style-guide.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  --timeline-text-color: rgba(61, 166, 255, 1);
+  --text-primary-color: rgba(255, 255, 255, 1);
+  --text-input-color: rgba(255, 255, 255, 1);
```
Comment (thuanpham2311) [inline docs/style-guide.md:26]: repeat color: text-primary-color === text-input-color -> delete: text-input-color

File: docs/style-guide.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  --highlight-line-bg-color: rgba(255, 255, 255, 0.1);
+  --heading-text-color: rgba(255, 241, 220, 1);
+  --hashtag-text-color: rgba(245, 160, 40, 1);
+  --forward-bg-color: rgba(44, 44, 44, 1);
+  --box-bg-color: rgba(72, 72, 72, 1);
+  --backward-bg-color: rgba(72, 72, 72, 1);
+}
+```
+
+## Typography
```
Comment (thuanpham2311) [inline docs/style-guide.md:43]: Add: css variable font size

File: docs/style-guide.md (hunk 2 above)
Comment (thuanpham2311) [reply docs/style-guide.md:26]: it doesn't really repeat, can be tried in the future

**Pull request merged**